docs/src/main/docs/grpc/client/02_configuration.adoc

```diff
@@ -0,0 +1,129 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+:javadoc-base-url-api: {javadoc-base-url}?io/helidon/grpc/server
+:pagename: grpc-client-configuration
+:description: Helidon gRPC Client Configuration
+:keywords: helidon, grpc, java, configuration
+
+= gRPC Client Configuration
+
+Configure the gRPC client using the Helidon configuration framework, either programmatically or via a configuration file.
+
+As mentioned earlier, creating a `GRPCServiceClient` involves:
+
+1. Creating a `ClientServiceDescriptor` which describes the methods in the service that this client can invoke.
+2. Creating a gRPC Channel through which the Client communicates with the server.
+
+Both of the above can be configured using the Helidon configuration framework, either programmatically
```

Comment:
This is not correct, neither `ClientServiceDescriptor` nor creating a `Channel` can be done with Helidon configuration.

Reply:
Removed.